app/examples/surfingkeys

```diff
@@ -0,0 +1,108 @@
+" Options
+set adblocker=update
+set devtoolsposition=split
+set downloadmethod=confirm
+set firefoxmode=always
+set fontsize=16
+set mintabwidth=250
+set norestoretabs
+set permissioncamera=ask
+set permissiondisplaycapture=allow
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set splitbelow
+set tabclosefocusright
+set tabreopenposition=previous
+set windowtitle=title
+set showcmd
```

Comment:
`showcmd` is enabled by default in Vieb so we can leave this out, aside from that your PR is great!

Reply:
I've removed it in the latest commit.